packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -646,11 +660,18 @@ class SliverFillViewport extends SliverMultiBoxAdaptorWidget {
   }
 }
 
+// This class exists to dissociate [KeepAlive] from [RenderSliverMultiBoxAdaptor].
+abstract class SliverWithKeepAliveElementMixin extends RenderObjectElement {
```

Comment:
Same as above - do we need this?  This class doesn't really add any behavior to `RenderObjectElement`, which we could just use instead.